lib/libutils/ext/include/util.h

```diff
@@ -92,6 +92,8 @@
 #define ALIGNMENT_IS_OK(p, type) \
 	(((uintptr_t)(p) & (__alignof__(type) - 1)) == 0)
 
+#define IS_ALIGNED(x,a) (!((x) & ((a) - 1)))
```

Comment:
I find weird that `ALIGNMENT_IS_OK()` expects a type argument and `IS_ALIGNED()` a byte size. @ldts, I think understand you didn't like having those 2 macros together.

I would propose to rename `ALIGNMENT_IS_OK(addr, type)` to `IS_ALIGNED_WITH_TYPE(addr, type)` or like and have a somewhat standard `IS_ALIGNED()` as you propose. Would prefer:
```c
#define IS_ALIGNED(x, a)		(!((x) & ((a) - 1)))
#define IS_ALIGNED_WITH_TYPE(x, type)	IS_ALIGNED((x), __alignof__(type))
```